### ISSUE TITLE
Interrupt mode?

### ISSUE THREAD

**Issue by chconnor** [alex-code/GT911#3]
Hello -- I've been trying to get this same screen to work in interrupt mode. Does your library work for that? You mentioned in the readme that other libraries use interrupt mode -- did you ever get anything from the interrupt pin?

Here's the code I am trying:
```
#include <Arduino.h>
#include <GT911.h>

GT911 ts = GT911();

void setup() {
  Serial.begin(115200);
  ts.begin(15,26);
}

void loop() {
  uint8_t touches = ts.touched(GT911_MODE_INTERRUPT);

  if (touches) {
    GTPoint* tp = ts.getPoints();
    for (uint8_t  i = 0; i < touches; i++) {
      Serial.printf("#%d  %d,%d s:%d\n", tp[i].trackId, tp[i].x, tp[i].y, tp[i].area);
    }
  }
}
```

This works fine in GT911_MODE_POLLING, but nothing happens in interrupt mode.

Reading through the docs/schematics, it seems like the GT911 INT line may only get to like 1.5V, and I'm wondering if that might have something to do with this conundrum. Thanks for any insight!

**Comment by chconnor** [comment]
Update: it would appear that the GT911 chip can be configured two ways with respect to the INT output levels. If VDDIO is tied to 3.3v, then INT outputs ~2.8V (==HIGH). But if VDDIO is left floating then it only outputs ~1.5V (==LOW). My display module also runs the INT signal through a level shifter, but AFAICT that shifter is expecting a real 3.3V-level HIGH and if it's not getting one then the output pin of my display module ends up as "LOW" all the time, not even a 1.5V that I could possible work with.

Long story short, if you got interrupt mode to work, please let me know, but I think these issues could be unique to my display module.
